File: codigos_de_prueba/prueba1.py
def compute_average(numbers_list):
    sum_values = 0
    count = 0
    for value in numbers_list:
        sum_values = sum_values + value
        count += 1
    if count == 0:
        return 0.0
    average = sum_values / count
    return average

def get_highest_number(values):
    highest = values[0]
    for current in values:
        if current > highest:
            highest = current
    return highest

data = [5, 8, 2, 10, 3]
print(f"Average value: {compute_average(data)}")
print(f"Highest number: {get_highest_number(data)}")

File: codigos_de_prueba/prueba1.cpp
#include <iostream>
#include <vector>

using std::cout;
using std::endl;
using std::vector;

float computeAverage(const vector<int>& data) {
    if (data.size() == 0) return 0.0f;
    
    float total = 0.0f;
    for (auto val : data) {
        total = total + val;
    }
    return total / data.size();
}

int findMaxValue(const vector<int>& values) {
    if (values.empty()) return -999;
    
    int current_max = values[0];
    for (size_t j = 0; j < values.size(); j++) {
        if (values[j] > current_max) {
            current_max = values[j];
        }
    }
    return current_max;
}

int main() {
    vector<int> numbers = {12, 45, 23, 67, 8, 34};
    
    cout << "Average: " << computeAverage(numbers) << endl;
    cout << "Max value: " << findMaxValue(numbers) << endl;
    
    return EXIT_SUCCESS;
}